spec/models/topic_embed_spec.rb

```diff
@@ -40,13 +40,22 @@
         expect(post.topic.category).to eq(embeddable_host.category)
       end
 
-      it "Supports updating the post" do
-        new_user = Fabricate(:user)
+      it "Supports updating the post title" do
+        post = TopicEmbed.import(user, url, "I am a new title", contents)
 
-        post = TopicEmbed.import(new_user, url, "I am a new title", "muhahaha new contents!")
+        expect(post.topic.title).to eq("I am a new title")
+      end
+
+      it "Supports updating the post content" do
+        post = TopicEmbed.import(user, url, title, "muhahaha new contents!")
 
         expect(post.cooked).to match(/new contents/)
-        expect(post.topic.title).to eq("I am a new title")
+      end
+
+      it "Supports updating the post author" do
+        new_user = Fabricate(:user)
+        post = TopicEmbed.import(new_user, url, title, contents)
+
         expect(post.user).to eq(new_user)
```

Comment:
@xrav3nz Hmm how was this assertion passing previously? I feel that something is not right because if this assertion was passing previously, this assertion is either wrong or we're missing another assertion.

Reply:
This passed because the post body changed in the setup before. (Previously, changes to author and title would only be recorded if the post body also changed)

However, you actually reminded me that I need another assertion here: `PostOwnerChanger` also changes the `post.topic.user` (if post is the first one), whereas `PostRevisor` doesn't. Added this and rebased!